### DISCUSSION TITLE
I can't submit proof materials.

### DISCUSSION THREAD

**Discussion by AreroKetahi** [community/community#110494] [GitHub Education] [answered]
### Hi everyone,

After I submit a screenshot of my grades to prove my in-school status, I received a rejection.

This is the report for my first request:
- If your school does not provide academic email then you must enable use of your device camera to capture your image.
- The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.
- Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
- You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. Your must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

The reason why I use VPN is I can't open GitHub smoothly.

In the second time, I turn off VPN, back to school and use school network directly requesting, everything work well, except I upload the proof materials, I use a transcript with my school logo and a clear date, that transcript also contains my legal name, student number and clear school address. But I failed, I was stuck in a cycle of not being able to submit, once I click the "Process my application" button, the web page automatically refresh and back to "Upload proof of your academic status" page, after I wait for a week, it still not work, I'm very confuse on that.

**Comment by samus-aran** [comment]
Hi @AreroKetahi 👋 

Is there any other network you can use close to your campus? When uploading evidence consider the image size keeping it smaller than 1MB if possible. Try and keep the text still legible.

**Comment by AreroKetahi** [comment, marked as answer]
I thought that I knew why I can't submit the images, because the document look too dark, I turned up the exposure and submit it again, and it pass.

**Comment by Amsb303** [comment]
I am also getting rejected but no reason is included in the info-box 
![Screenshot 2024-03-04 224106](https://github.com/community/community/assets/149975746/4f29cfd7-9454-4507-8bc2-78b43165e714)
